llarp/net.cpp

```diff
@@ -887,6 +887,61 @@ namespace llarp
     return found;
   }
 
+  std::string
+  findFreePrivateRange()
+  {
+    // pick ip
+    struct privatesInUse ifsInUse = llarp_getPrivateIfs();
```

Comment:
does `llarp_getPrivateIfs` iterate over all of our network interfaces? i don't understand the `findFreePrivateRange` function.

Reply:
Yes, it looks at all interfaces for any private IPs bound to them. findFreePrivateRange finds a range that you're not currently using. I think we may have talked a bit more about this in chat. Let me know if you have any more questions about it.